NineChronicles.DataProvider/RenderSubscriber.cs

```diff
@@ -1210,13 +1264,13 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                                 }
                             }
 
-                            _avatarList.Add(AvatarData.GetAvatarInfo(ev.OutputState, ev.Signer, ev.Action.AvatarAddress, ev.Action.RuneInfos, _blockTimeOffset));
+                            _avatarList.Add(AvatarData.GetAvatarInfo(outputState, ev.Signer, ev.Action.AvatarAddress, ev.Action.RuneInfos, _blockTimeOffset));
 
                             int raidId = 0;
                             bool found = false;
                             for (int i = 0; i < 99; i++)
                             {
-                                if (ev.OutputState.Delta.UpdatedAddresses.Contains(
+                                if (outputState.Delta.UpdatedAddresses.Contains(
```

Comment:
Does it contain real `Delta.UpdatedAddresses`? Don't we need `ITrie.Diff()` in this case?

Reply:
See https://github.com/planetarium/NineChronicles.DataProvider/pull/515/commits/9aa4c9ea78c7df731fb04b0946d11029db58ef36.

- Possibly a lot slower than before.
- Behavior might not be the same as diff of `ITrie` only searches for **actual** changed value.